src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/ReadyToRunReader.cs

```diff
@@ -200,6 +200,9 @@ public TargetArchitecture TargetArchitecture
                     case Machine.Arm64:
                         return TargetArchitecture.ARM64;
 
+                    case Machine.LoongArch64:
```

Comment:
The following compling error:
```
src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/ReadyToRunReader.cs(203,34): error CS1061: (NETCORE_ENGINEERING_TELEMETRY=Build) 'Machine' does not contain a definition for 'LoongArch64' and no accessible extension method 'LoongArch64' accepting a first argument of type 'Machine' could be found (are you missing a using directive or an assembly reference?)
```

Is it for the dotnet-SDK which not supporting LoongArch64 ?
While I had compile successfully on my LoongArch64-linux, liking
```
qiao@3a5000-213:~/work_qiao/bk_runtime_loongarch/src/coreclr/tools/aot/crossgen2$ ~/sdk-release6.0.102-v2/dotnet build crossgen2.csproj 
用于 .NET 的 Microsoft (R) 生成引擎版本 17.0.0+c9eb9dd64
版权所有(C) Microsoft Corporation。保留所有权利。

  Determining projects to restore...
  All projects are up-to-date for restore.
  ILCompiler.DependencyAnalysisFramework -> /home/qiao/work_qiao/bk_runtime_loongarch/artifacts/bin/ILCompiler.DependencyAnalysisFramework/loongarch64/Debug/ILCompiler.DependencyAnalysisFramework.dll
  ILCompiler.TypeSystem.ReadyToRun -> /home/qiao/work_qiao/bk_runtime_loongarch/artifacts/bin/ILCompiler.TypeSystem.ReadyToRun/loongarch64/Debug/ILCompiler.TypeSystem.ReadyToRun.dll
  ILCompiler.Diagnostics -> /home/qiao/work_qiao/bk_runtime_loongarch/artifacts/bin/ILCompiler.Diagnostics/loongarch64/Debug/ILCompiler.Diagnostics.dll
  ILCompiler.ReadyToRun -> /home/qiao/work_qiao/bk_runtime_loongarch/artifacts/bin/ILCompiler.ReadyToRun/loongarch64/Debug/ILCompiler.ReadyToRun.dll
  crossgen2 -> /home/qiao/work_qiao/bk_runtime_loongarch/artifacts/bin/coreclr/Linux.loongarch64.Debug/crossgen2/crossgen2.dll

已成功生成。
    0 个警告
    0 个错误

已用时间 00:00:36.27
````

Reply:
To fix this. you would need to bump SystemReflectionMetadataVersion in eng\Versions.props to a version that has the addition of `Machine.LoongArch64` (ie 7.0.0-preview.2... - same as what some other versions have). Do you happen to have this version bump in your local enlistment?

It would be best to submit this version bump as a separate PR. These version bumps often end up with some unexpected issues and it would be best to deal with them separately.

Reply:
> To fix this. you would need to bump SystemReflectionMetadataVersion in eng\Versions.props to a version that has the addition of `Machine.LoongArch64` (ie 7.0.0-preview.2... - same as what some other versions have). Do you happen to have this version bump in your local enlistment?

Yes, we have back-port the LoongArch64 to the sdk6.0 in our local enlistment.
 
> It would be best to submit this version bump as a separate PR. These version bumps often end up with some unexpected issues and it would be best to deal with them separately.

OK, I will submit it separately.